lib/api/controllers/collectionController.js

```diff
@@ -413,6 +425,7 @@ function validateSpecificationList (validator, list) {
       return {
         valid: false,
         details: errors,
+        // @todo useless, to be removed in a future version
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)